include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -340,9 +340,10 @@ class BlmcJointModules
           motors,
         const Vector& motor_constants,
         const Vector& gear_ratios,
-        const Vector& zero_angles)
+        const Vector& zero_angles,
```

Comment:
This will most likely not even compile in your machine right?
I mean we use the BlmcJointModules in the blmc_robots solo and teststand as well.
So we should fix them before landing this PR.
Otherwise I approve this PR.

Reply:
Sorry I confused `blmc_joint_moduleS` with the one without `s`.